### PR TITLE
updated all the URLs

### LINKED ISSUE #2: [Feature 🚀]: URL Links
### Contact Details

yashdev.yvd@gmail.com

### What is the expected behavior?

As this Docusaurus is built by Facebook so all url links are are directing to there website.

### What is the current behavior?

After updating the path all urls will direct the viewer to our DeveloperDAO pages.

### Reproduction steps

```bash
1.
2.
3.
...
```

### Version

_No response_

### Relevant log output

_No response_

### Your Environment

* Version used:
* Environment name and version (e.g. Chrome 39, node.js 5.4):
* Operating System and version (desktop or mobile):
* Link to your project:

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by yashdev9274**
# Description of PR... 

## Changes In URLs at Footer 

- Twitter 
- Newsletter
- Discord 
- Github

List the issues that this fixes or connects to. For example:
Fixes #2 Issue opened by me
Will close issue #2  on merge

### CODE REVIEW

File: docusaurus.config.js [hunk 1]
```diff
@@ -95,24 +95,25 @@ const config = {
             items: [
               {
                 label: "Docs",
-                to: "/",
+                to: "https://docs-one-pi.vercel.app/",
```
Comment (kempsterrrr) [inline docusaurus.config.js:98]: update to - docs.developerdao.com (not live yet but will be final address)

**Review by kempsterrrr — COMMENTED** [review]
Nice work @yashdev9274 - looking good just left a comment to update the URL to the final URL when these docs are finished :)

File: docusaurus.config.js (hunk 1 above)
Comment (yashdev9274) [reply docusaurus.config.js:98]: Yeah sure, I'm on it!

**Review by kempsterrrr — APPROVED** [review]
Looks good!

**Pull request merged**